Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -460,9 +468,19 @@ int Sequencer::run() {
     }
   }
 
-  // execute the parallel event loop
+  // loop preparation
   std::atomic<std::size_t> nProcessedEvents = 0;
   std::size_t nTotalEvents = eventsRange.second - eventsRange.first;
+
+  // prepare event timing data
+  bool enableEventTiming = m_cfg.outputEventTimingFile.has_value();
+  std::size_t nEventTimingData = 0;
+  if (enableEventTiming) {
+    nEventTimingData = nTotalEvents * names.size();
+  }
+  std::vector<EventTimingInfo> eventTimingData(nEventTimingData);
```

Comment:
```suggestion
  std::vector<EventTimingInfo> eventTimingData;
  if (enableEventTiming) {
    eventTimingData.resize(nTotalEvents * names.size());
  }
```

matter of taste but to me it looks a bit simpler